docs/migrate-github-radicle.mdx

```diff
@@ -0,0 +1,141 @@
+- Continue pushing changes to Radicle alone, or mirror changes in both Radicle and GitHub.
+
+This guide also works if you're currently using GitLab, Gitea, or any other Git-based code hosting platform.
+
+:::info
+
+Radicle doesn't yet have feature parity with GitHub&mdash;we're still working on visual code reviews, an issue board,
+and CI/CD pipelines. Even you _need_ these features, stopping you from migrating in full, we still encourage you to
+migrate your project to Radicle and mirror your changes so that you can quickly make the switch when the time is right!
+
+:::
+
+## Install the Radicle CLI
+
+To migrate from GitHub to Radicle, you first need to install the Radicle CLI and create a Radicle identity, which is
+like creating an account on GitHub. It's how your projects are identified on the Radicle network, and it's your key to
+hosting code and collaborating with others on your project.
+
+<Installation />
+
+Once you're done with the first two steps&mdash;**Install the Radicle `rad` CLI.** and **Run rad auth to create your
+Radicle identity.**&mdash;you can return here for the rest of the instructions.
+
+## Initialize your project on Radicle
+
+Radicle wraps around the `git` binary and processes you already use to manage your project's data and history. This
+rapper lets you push your code to the sovereign Radicle network and collaborate with others.
+
+Because you already have a project you've pushed to GitHub, you already initialized the Git repository with `git init`,
+which created the `.git` directory to store objects, heads, refs, and more.
+
+Initializing a project on Radicle is as simple as setting up Git for the first time:
+
+```
+cd your-project-directory/
+rad init
+```
+
+Add a description and choose your default branch, which is typically either `main` or `master`.
+
+Running `rad init` does two things:
+
+- Creates a unique peer-to-peer identity&mdash;a project URN&mdash;for your project.
+- Adds a new `remote` to your project's existing configuration using that new identity, for pushing code to a unique
+  address on the Radicle network.
+
+You'll also see a line similar to `ok Project initialized: rad:git:hnrkqi6ohci9m59i54ppiy3fqkedkjt98ymdo`. The text
+after `rad:git:` is your project's URN.
+
+You can always find your project's URN by running `rad .` from the project's directory.
+
+## Push to the Radicle network
+
+You can push to the network now that you've initialized your project on Radicle. Assuming you have no unstaged changes
+in your repository, you're ready to push.
+
+```
+rad push
+```
+
+Since this is your first push to Radicle, the CLI asks which seed node you want to sync with. Radicle provides three
+seed nodes with _identical_ functionality&mdash;`pine`, `willow`, and `maple`.
+
+Choose wisely!
+
+:::tip
+
+You can also sync to multiple Radicle-hosted seed nodes, or [host your own seed
+node](https://github.com/radicle-dev/radicle-client-services), if you'd like additional resiliency or security.
```

Comment:
With the caveat that the data will need to be updated on each of them individually.